etl/steps/data/garden/ggdc/2020-10-01/ggdc_maddison_project_database.py

```diff
@@ -0,0 +1,245 @@
+"""Load dataset for the Maddison Project Database from walden, process it, and transfer it to garden.
+
+Current dataset assumes the following approximate mappings:
+* "U.R. of Tanzania: Mainland" -> "Tanzania" (ignoring Zanzibar).
+* "Sudan (Former)" -> "Sudan" (including South Sudan).
+
+Definitions according to the Notes in the data file:
+* "gdppc": Real GDP per capita in 2011$.
+* "pop": Population, mid-year (thousands).
+
+"""
+
+import json
+from typing import cast, Dict
+
+import pandas as pd
+from owid.catalog import Dataset, Table
+from owid.walden import Catalog
+
+from etl.paths import STEP_DIR
+from etl.steps.data.converters import convert_walden_metadata
+
+# Institution name.
+NAMESPACE = "ggdc"
+# Dataset name.
+DATASET_NAME = "ggdc_maddison_project_database"
+# Original dataset publication date.
+VERSION = "2020-10-01"
+# Column name for GDP in output dataset.
+GDP_COLUMN = "GDP"
+# Column name for GDP per capita in output dataset.
+GDP_PER_CAPITA_COLUMN = "GDP per capita"
+
+
+def load_countries() -> Dict[str, str]:
```

Comment:
This would make a nice helper function, how about moving it to https://github.com/owid/etl/blob/master/etl/data_helpers.py so that I can reuse it for other projects? :)
It could have path as an input parameter (so that one could use `Path(__file__).parent` instead of specifying the full path)

Reply:
For now I'm going to keep it here, but feel free to move the function to data_helpers if you think that's appropriate (but let's leave it for a different branch).